apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,65 @@
+import { PurchaseOrder } from '../interfaces';
+import Link from 'next/link';
+
+const fetchPurchaseOrders = async (sortField: string, sortOrder: string): Promise<PurchaseOrder[]> => {
+  const res = await fetch(`http://localhost:3100/api/purchase-orders?sortField=${sortField}&sortOrder=${sortOrder}`, { cache: 'no-cache' });
+  return res.json();
+};
+
+const PurchaseOrdersPage = async ({ searchParams }: { searchParams: { sortField?: string, sortOrder?: string } }) => {
+  // Implementing server side sorting to keep the code clean and since there aren't a lot of records, the performance hit is not noticable.
```

Comment:
Wanted to highlight my decision to use server-side sorting. I can easily refactor to use client-side sorting if y'all think that the ability for the page to be more performant on larger datasets is more important than the relative simplicity of server-side sorting.

Reply:
What do you mean by client-side sorting? Using a client component? Or sorting in JavaScript?  How would that work if we were to introduce pagination?

Reply:
Sorry, I meant using a client component and not that I think about it again, you don't gain any performance by doing so and the code just becomes unnecessarily complicated with react hooks and state management.